omp/__init__.py

```diff
@@ -18,6 +16,70 @@ def compiler():
 cxx = compiler()
 
 
+def get_shared_lib_extension(is_python_ext=False):
```

Comment:
Just in case: Is this function extracted from another codebase? If so, we should mention it somewhere.

Reply:
yes, from `numpy.distutils`. I'll mention it.